### ISSUE TITLE
Does SSHFS works with Open-SSH on Windows when using key-pair auth?

### ISSUE THREAD

**Issue by Yanivfd** [libfuse/sshfs#281]
I mean, can I save the private key in SSH-Agnet of Open-SSH and then mount a remote folder with the private key saved in Open-SSH?

**Comment by medea-learner** [comment]
> I mean, can I save the private key in SSH-Agnet of Open-SSH and then mount a remote folder with the private key saved in Open-SSH?

As a solution y can try it yourself and then conclude :)

**Comment by z-tb** [comment]
Yeah, this works on Windows 10 if the OpenSSH server "Optional" software package is installed and some [extra](https://learn.microsoft.com/en-us/windows-server/administration/openssh/openssh_keymanagement) configuration is done for [OpenSSH on the windows host](https://stackoverflow.com/questions/16212816/setting-up-openssh-for-windows-using-public-key-authentication). This was done an a windows 10 EC2 but that really shouldn't matter. I have no idea if this works on windows 11 the same way, or at all.

Run the powershell commands as Admin
![Image](https://github.com/user-attachments/assets/cad97a60-57be-4731-9551-532b2ad43cfb)

Don't login as admin over ssh. create a user and login from the remote system with the password:
![Image](https://github.com/user-attachments/assets/b977763b-70e6-44b4-8f1f-8f48c6786482)

Then create the `.ssh` directory for the windows user and drop the **pub** key of the remote user into `authorized_keys`. The `sftp` command or `ssh-copy-id` could be used too. This is just my habit:
![Image](https://github.com/user-attachments/assets/23c83df6-c19b-41b8-9ee1-0d7f94add1d5)

The next part is important to get right or the permissions on windows will fight with you the whole way. Follow the StackOverflow link above at the answer given by `n0rd`.  Step 6-10 involve a lot but must be followed exactly:

![Image](https://github.com/user-attachments/assets/b599dfca-bb26-4278-89c0-0d9720c54063)

Once that is all configured correctly, you can mount the remote `C:` drive like this:

![Image](https://github.com/user-attachments/assets/15a5227f-3488-44da-a833-9a9e16f1358b)
